README.md

```diff
@@ -411,35 +411,41 @@ nativeAlert: ({
 
 ### nativeMessage
 
-Show a native message dialog. Use it to display feedback messages.
+<kbd>App version >=14.10 `withDismiss`, `duration` and `action` in
+response</kbd>.
+
+Show a native snackbar message. Use it to display feedback messages.
 
 If the bridge is not present (eg. showing the page in browser), fallbacks to a
 browser alert.
 
--   `buttonText` property is ignored in iOS.
-
 <img height="550" src="doc/webview-bridge-message-ios.png"><img height="550" src="doc/webview-bridge-message-android.png">
 
 ```ts
 nativeMessage: ({
         message: string;
-        duration?: number; // milliseconds
-        buttonText?: string; // Android only
+        duration?: 'PERSISTENT' | 'FIVE_SECONDS' | 'TEN_SECONDS';
```

Comment:
isn't this a breaking change?

Reply:
Ok, I see that the apps were ignoring this param but this isn't implementing the proposed solution in #117

![image](https://github.com/Telefonica/webview-bridge/assets/579417/a81d07ff-0d71-493d-a3c0-b0a06887fa19)

Reply:
Ok, I see that the implementation allows the `number` type. Which one is correct?